senders/telegram/handle_message.go

```diff
@@ -29,20 +29,22 @@ func (sender *Sender) getResponseMessage(message *telebot.Message) (string, erro
 		if message.Chat.Username == "" {
 			return "Username is empty. Please add username in Telegram.", nil
 		}
-		err := sender.DataBase.SetUsernameID(messenger, "@"+message.Chat.Username, chatID)
+		_, err := sender.setChat(message)
 		if err != nil {
 			return "", err
 		}
 		return fmt.Sprintf("Okay, %s, your id is %s", strings.Trim(fmt.Sprintf("%s %s", message.Sender.FirstName, message.Sender.LastName), " "), chatID), nil
-	case message.Chat.Type == telebot.ChatSuperGroup || message.Chat.Type == telebot.ChatGroup:
-		err := sender.DataBase.SetUsernameID(messenger, message.Chat.Title, chatID)
+	case (message.Chat.Type == telebot.ChatSuperGroup || message.Chat.Type == telebot.ChatGroup) && strings.HasPrefix(message.Text, "/start"):
+		contactValue, err := sender.getContactValueByMessage(message)
 		if err != nil {
-			return "", err
+			return "", fmt.Errorf("failed to get contact value from message: %s", err.Error())
```

Comment:
It is better to wrap errors using %w

Reply:
Done https://github.com/moira-alert/moira/pull/1022/commits/ebf30d12e8e97f8615b6bf842f27245e0aa05520